common4j/src/main/com/microsoft/identity/common/java/exception/ClientException.java

```diff
@@ -27,6 +27,12 @@
     public static final String sName = ClientException.class.getName();
     private static final long serialVersionUID = -2318746536590284648L;
 
+    /**
+     * Indicates that a request was made for a specific environment, and this information was not
+     * present on the device.
```

Comment:
not present on the device? Is this specifically not present in broker account data in Account Manager? (or IBrokerAccountDataStorage)

Reply:
Specifically, there's no PRT present for the cloud we asked for.